etc/octavia_proxy.conf

```diff
@@ -50,6 +51,7 @@ auth_strategy = validatetoken
 # A comma separated list of dictionaries of the enabled provider driver names
 # and descriptions.
 enabled_provider_drivers = elbv2:The Open Telekom Cloud Enhanced LB driver.,elbv3:The Open Telekom Cloud AZ aware LD driver.
+load_balancer_endpoint_override = url:http://127.0.0.1:9876/,var1:variable1,var1:variable2
```

Comment:
why `var1:variable1`. I would expect something similar to how service catalog work: `http://127.0.0.1:9876/vX.Y/%(project_id)s`